.github/workflows/scripts/sensitive_file_check.py

```
@@ -0,0 +1,232 @@
+    """Get the list of changed files between base and head commits.
+
+    Args:
+        base_sha: The base commit SHA.
+        head_sha: The head commit SHA.
+
+    Returns:
+        result: List of changed file paths.
+
+    """
+    try:
+        output = subprocess.check_output(
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)